core/shared/src/main/scala/org/threeten/bp/format/DateTimeParseException.scala

```diff
@@ -56,9 +56,12 @@ class DateTimeParseException(
   message:                String,
   parsedData:             CharSequence,
   private val errorIndex: Int,
-  cause:                  Throwable = null
+  cause:                  Throwable
 ) extends DateTimeException(message, cause) {
 
+  def this(message: String, parsedData: CharSequence, errorIndex: Int) =
+    this(message, parsedData, errorIndex, null)
```

Comment:
You can't just be lazy and slap a `= null` 😛  it doesn't generate a binary-signature that is compatible with the JDK.